DEHPCommon/HubController/HubController.cs

```diff
@@ -662,5 +662,38 @@ public async Task Download(FileRevision fileRevision, FileStream destination)
         /// </summary>
         public IEnumerable<ExternalIdentifierMap> AvailableExternalIdentifierMap(string toolName)
             => this.OpenIteration.ExternalIdentifierMap.Where(x => x.ExternalToolName == toolName);
+
+        /// <summary>
+        /// Adds a new <see cref="ModelLogEntry"/> record to the <see cref="EngineeringModel.LogEntry"/> list of the current <see cref="OpenIteration"/> and registers the change to a <see cref="ThingTransaction"/>
+        /// </summary>
+        /// <param name="content">The value that will be set to the <see cref="ModelLogEntry.Content"/></param>
+        /// <param name="transaction">The <see cref="ThingTransaction"/> that will get the changes registered to</param>
+        public void RegisterNewLogEntryToTransaction(string content, ThingTransaction transaction)
+        {
+            if (string.IsNullOrWhiteSpace(content))
```

Comment:
please allow people to transfer with no message. But if its empty simply skip the rest (return)